static/panes/pane.ts

```diff
@@ -116,6 +118,9 @@ export abstract class Pane<S> {
      */
     abstract registerOpeningAnalyticsEvent(): void;
 
+    /** Optional overridable code for initalizing necessary elements before rest of registers **/
+    registerPreElements(state: S): void {}
```

Comment:
The one thing I dislike is that I had to define a new function that gets called before all the registers so that you have a chance to initialize class atributes that need the state, but that are not buttons. If someone comes up with a better approach it would be great, but this also works :)

Reply:
Maybe something like registerDynamicElements?

Reply:
Yup, much better, thanks